src/main/java/com.bbd.bursary.manager/Repository/JdbcInstituteRepository.java

```diff
@@ -0,0 +1,33 @@
+package com.bbd.bursary.manager.Repository;
+
+import com.bbd.bursary.manager.Model.Institute;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class JdbcInstituteRepository implements InstituteInterface {
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public int save(Institute institute) {
+        return jdbcTemplate.update("INSERT INTO Institute_Info (InstituteName, ContactDetailsID, BBDBursaryInstituteStatus) VALUES (?,?,?)",
```

Comment:
With inserting contactDetails Id it leads to nowhere. These were the procedures that I asked if I must create. In order to add an institute we add values into a few tables so you cant just insert into a single table😂

Reply:
ok thats why i suggested a meeting